pkg/addons/addons.go

```diff
@@ -57,9 +62,23 @@ func readDB(cs *acsapi.ContainerService, c *config.Config) (map[string]unstructu
 			return nil, err
 		}
 
+		azs, err := GetStorageAccountsClient(c)
```

Comment:
run this outside of the loop?

Reply:
Or maybe move this code altogether in newClient. I think it makes more sense in there.

Reply:
Where do you recommend this go?  As part of the client instantiation?  As a var inside of the addons.go?  Inside of addons.go main?

Reply:
it's client-related code so client instantiation would be my choice.